test/checkpoint/test_checkpoint.c

```diff
@@ -113,13 +113,37 @@ main(int argc, char *argv[])
     testutil_parse_begin_opt(argc, argv, SHARED_PARSE_OPTIONS, &g.opts);
 
     while ((ch = __wt_getopt(
-              progname, argc, argv, "C:c:Dk:l:mn:pr:Rs:S:T:t:vW:xX" SHARED_PARSE_OPTIONS)) != EOF)
+              progname, argc, argv, "C:c:d:Dk:l:mn:pr:Rs:S:T:t:vW:xX" SHARED_PARSE_OPTIONS)) != EOF)
         switch (ch) {
         case 'c':
             g.checkpoint_name = __wt_optarg;
             break;
         case 'C': /* wiredtiger_open config */
-            config_open = __wt_optarg;
+            strcpy(config_open, __wt_optarg);
+            break;
+        case 'd': /* disaggregated storage options */
```

Comment:
I believe we should only set `g.opts.disagg_storage = true;` here, and then worry about individual fields in an abstracted function called later? What do you think?

Reply:
Removed to dedicated function.